gds-api-adapters.gemspec

```diff
@@ -23,6 +23,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'lrucache', '~> 0.1.1'
   s.add_dependency 'rest-client', '~> 1.8.0'
   s.add_dependency 'rack-cache'
+  s.add_dependency 'uk_postcode', '2.1.0'
```

Comment:
The approach taken by `govuk_message_queue_consumer` is that if Airbrake is available, cool, we'll tell it about an error, if not though that's cool too we just won't do anything.  We could take the same approach - if the uk_postcode gem is available we'll sanitize the postcode, if not, we won't, but I'm not sure that gives us much benefit.

The ultimate problem here is that no matter what we do about making the dependency optional (required to use mapit at all or required only if you want to sanitize postcodes) we have to put  `gem 'uk_postcode'` into the Gemfile of every app that wants to talk to mapit about postcodes, otherwise we don't get the benefit that the `PostcodeSanitizer` is there to provide - and that seems a bit onerous.